cerebrium/getting-started/introduction.mdx

```diff
@@ -1,40 +1,44 @@
-  to worry about CPUs/GPUs, Kubernetes, queues, monitoring, scaling etc. We take
-  care of this to create a robust and seamless developer experience.
-- <b>
-    <u>Research</u>
-  </b>
-  We try to implement the latest research towards your model as best we can in
-  order for you to deliver the best experience to your users. Besides giving you
-  to the option to select the best chip for your workload, we look to see how we
-  can take maximum advantage of the GPU to get your model to run faster and
-  cheaper without sacrificing performance.
-
-## Our users favorite features
-
-- \<5 second cold-start times
-- Wide variety of GPUs
-- Automatic scaling from 1 to 10k requests in seconds
-- Define pip/conda container environments in code
-- Secrets manager
-- One-click deploys
-- Persistent Storage
-
-<b>All of this in just a few lines of code!</b>
-
-To get started, head to our [installation page](/installation)
+Cerebrium is a cloud GPU infrastructure platform that transforms how teams deploy and scale machine learning applications. By handling the complex infrastructure – GPUs, Kubernetes, queues, monitoring, and scaling – Cerebrium lets teams focus on building applications that deliver value.
+
+The platform achieves this through rapid cold starts (Less than 5 seconds), flexible GPU options, automated scaling (1 to 10,000+ requests), and simple deployment processes. With 99.9% uptime and continuous weekly improvements based on user feedback, Cerebrium provides the reliability needed for production ML apps.
+
+## Getting Started
+
+Setting up and deploying an app on Cerebrium takes just a few steps:
+
+### 1. Install the CLI
+
+```bash
+pip install cerebrium
+cerebrium login  # Redirects to your browser for authentication
+```
+
+### 2. Initialize a Project
+
+```bash
+cerebrium init my-first-app
+cd my-first-app
+```
+
+This creates a basic project with `main.py` for app code and `cerebrium.toml` for configuration.
+
+### 3. Deploy an App
+
+```bash
+cerebrium deploy
+```
+
+The app now builds and deploys, typically within a few seconds. Check out a detailed description of the deployment process for more information. Once deployed, an application becomes callable through an endpoint `https://api.cortex.cerebrium.ai/v4/{project-id}/{app-name}/{function-name}`.
+
+## How It Works
+
+Cerebrium uses containerization to ensure consistent environments and reliable scaling for apps. When code is deployed, Cerebrium packages it with all necessary dependencies into a container image. This image serves as a blueprint for creating instances that handle incoming requests. The system automatically manages scaling, creating new instances when traffic increases and removing them during quiet periods.
+
+For a detailed explanation of how Cerebrium builds and manages container images, see our [Defining Container Images Guide](/cerebrium/container-images/defining-container-images).
```

Comment:
Somewhere we need to mention Content-Aware-Storage - perhaps a section about What makes Cerebrium so fast?